docs/workforce-management-framework/topics/con-KeycloakVsPassport.adoc

```diff
@@ -0,0 +1,37 @@
+[]
```

Comment:
This adoc looks like an introduction to Raincather security which explains the two options available. Perhaps 'con-raincatcher-security',  'con-security-introduction'  or something similar for the module name?